.github/workflows/deploy-book-ghpages.yml

```diff
@@ -16,6 +16,20 @@ on:
     # - some-subfolder/**
   workflow_dispatch:
 
+# Inherit configuration variables in the environment, or assign default values.
+# Configuration variables may be set as explained here:
+# https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository
+env:
+  # `publish` as primary by default
+  PRIMARY_BRANCH: ${{ vars.PRIMARY_BRANCH != '' && vars.PRIMARY_BRANCH || 'publish' }}
+  # Space-separated list of alias-rules, e.g. 'draft:main alias:really-long-branch-name'
+  # By default, `draft` links to `main`.
+  # If no aliases are wanted, BRANCH_ALIASES may be set to ' ' (space).
+  BRANCH_ALIASES: ${{ vars.BRANCH_ALIASES != '' && vars.BRANCH_ALIASES || 'draft:main book:publish' }}
+  # Space-separated list of branch names, e.g. 'main second third'.
+  # By default, deploy all branches. This is indicated by '*'.
+  BRANCHES_TO_DEPLOY: ${{ vars.BRANCHES_TO_DEPLOY != '' && vars.BRANCHES_TO_DEPLOY || '*' }}
```

Comment:
Do we want to deploy all branches by default?

Reply:
I would say yes... but maybe I don't have an overview of the implementations of that.